consensus/src/agreement/verifiers.rs

```diff
@@ -110,9 +151,18 @@ pub async fn verify_votes(
 
         let sub_committee = guard.intersect(bitset, cfg);
         let target_quorum = guard.quorum(cfg);
-
-        if guard.total_occurrences(&sub_committee, cfg) < target_quorum {
-            Err(Error::VoteSetTooSmall)
+        let total = guard.total_occurrences(&sub_committee, cfg);
+
+        if total < target_quorum {
+            tracing::error!(
+                desc = "vote_set_too_small",
+                committee = format!("{:#?}", sub_committee),
+                cfg = format!("{:#?}", cfg),
+                bitset = bitset,
+                target_quorum = target_quorum,
+                total = total,
```

Comment:
Ditto

Reply:
Indeed, this will be also covered with next issue.